modelcenter/PP-LCNetV2/introduction_en.ipynb

```diff
@@ -0,0 +1,239 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. PP-LCNetv2 Introduction\n",
+    "PP-LCNetv2 series model is a lightweight convolution neural network model proposed by PaddleCV team. This series model is improved based on PP-LCNetv1 and focuses more on higher model performance. The accuracy of this model can reach 77.04% on the ImageNet1k dataset. Meanwhile, the inference delay is about 4.32ms. In addition, considering the extensive use of OpenVINO reasoning framework in the industry, PP-LCNetv2 series model is specially optimized for Intel CPU combined with OpenVINO.\n",
+    "\n",
+    "The PP-LCNetv2 series model was developed and trained based on PaddleClas. For more information on PP-LCNet series models, see [PaddleClas-PPLCNetv2](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/en/models/PP-LCNetV2_en.md)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Performance\n",
+    "\n",
+    "The evaluation indexes of PP-LCNetV2 series models in ImageNet1k data set are shown as follows:\n",
```

Comment:
The evaluation indexes of PP-LCNetV2 series models in ImageNet1k data set are shown as follows  -> The evaluation results of PP-LCNetV2 series models on ImageNet1k dataset are shown as follows